src/Arcus.BackgroundJobs.AzureActiveDirectory/ClientSecretExpirationJob.cs

```diff
@@ -104,18 +121,27 @@ private ClientSecretExpirationEventType DetermineExpirationEventType(AzureApplic
                 { "RemainingValidDays", application.RemainingValidDays }
             };
 
-            var eventType = ClientSecretExpirationEventType.ClientSecretAboutToExpire;
-            if (application.RemainingValidDays < 0)
+            switch (eventType)
             {
-                eventType = ClientSecretExpirationEventType.ClientSecretExpired;
-                _logger.LogEvent($"The secret {application.KeyId} for Azure Active Directory application {application.Name} has expired.", telemetryContext);
-            }
-            else
-            {
-                _logger.LogEvent($"The secret {application.KeyId} for Azure Active Directory application {application.Name} will expire within {application.RemainingValidDays} days.", telemetryContext);
+                case ClientSecretExpirationEventType.ClientSecretExpired:
+                    telemetryContext["Description"] = $"The secret {application.KeyId} for Azure Active Directory application {application.Name} has expired.";
+                    _logger.LogSecurityEvent("Expired Azure Active Directory application secret", telemetryContext);
+                    break;
+                
+                case ClientSecretExpirationEventType.ClientSecretAboutToExpire:
+                    telemetryContext["Description"] = $"The secret {application.KeyId} for Azure Active Directory application {application.Name} will expire within {application.RemainingValidDays} days.";
+                    _logger.LogEvent("Soon expired Azure Active Directory application secret", telemetryContext);
```

Comment:
Why is this a regular event, whereas on line 128 it is logged as a security event ?

Reply:
Yeah, was thinking that there may be a different thing between 'expired' and 'soon expired'. Seems like the first one is more critical, but yeah, we could consider them both a security event. Let's change that!

Reply:
Done in https://github.com/arcus-azure/arcus.backgroundjobs/pull/139/commits/970be1ed4a5b1297589eec49711fd87551b252fa